### ISSUE TITLE
Request: Carthage support

### ISSUE THREAD

**Issue by danieljacobarcher** [matthewpalmer/Locksmith#21]
Requesting support for installing Locksmith via Carthage.

**Comment by matthewpalmer** [comment]
Ooft. Sorry I didn't reply to this earlier! Do you know of any good guides for setting this up?

**Comment by Jachobsen** [comment]
See "Supporting Carthage for your framework" in the main README of https://github.com/Carthage/Carthage

**Comment by Jachobsen** [comment]
:sparkles:

**Comment by larslockefeer** [comment]
Note that as of yet, you need to use `github "matthewPalmer/Locksmith" "04c6084a8f452f71813a9db7da5928ce8bb02ee3"` or `github "matthewPalmer/Locksmith" "master"` for Carthage to be able to checkout the supported version.

@matthewpalmer Do you plan on releasing a new version soon? Let me know if you need help with this?

**Comment by matthewpalmer** [comment]
@larslockefeer hoping to release a new version soon. Ran into a couple of strange Xcode issues though. Can you please do me a quick favour? If you clone this repo and run pod install on the demo app, and then try to run the tests, do they compile and run for you? (I'm getting an `ld: file not found` error.) Thanks

**Comment by larslockefeer** [comment]
@matthewpalmer It does not work for me as well, getting the same error.

I saw a similar error when using carthage to add Locksmith to my project, which seemed to be related to the fact that there was no `Info.plist`. Adding this file to the Locksmith project fixed it.

However, when I clone my own fork that includes the plist file, running `pod install` on the demo app and then trying to run the tests still doesn't work for cocoapods. I did manage to get it working in an isolated project though, for both carthage and cocoa pods, see pull request #34 for details.
